### PR TITLE
Import txt

### PR DESCRIPTION
**Pull request by zepdon**
-Now can import Scenes Option and Event from txt file
-Dialogue Slowly appear

### CODE REVIEW

File: dialogueMaker/dialogueMaker.cpp [hunk 1]
```diff
@@ -11,9 +12,16 @@ void Scene::printScene() {
   /*for (int i = 0; i < sceneLength; i++) {
     std::cout << "-";
   }*/
-  std::cout << "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------";
-  std::cout << '\n' << dialogue << '\n';
-  std::cout << "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------";
+  std::cout << "---------------------------------------------------------------------------------------------------------------------------";
+  //std::cout << '\n' << dialogue << '\n';
+  std::cout<< '\n';
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.cpp]:
```suggestion
  std::cout << "---------------------------------------------------------------------------------------------------------------------------\n";
```

File: dialogueMaker/dialogueMaker.cpp [hunk 2]
```diff
@@ -11,9 +12,16 @@ void Scene::printScene() {
...
+  for (int i = 0; i < dialogue.size(); i++)
+  {
+    std::cout << dialogue[i];
+    Sleep(1);
+  }
+  std::cout << '\n';
+  std::cout << "---------------------------------------------------------------------------------------------------------------------------";
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.cpp:24]:
```suggestion
  std::cout << "\n---------------------------------------------------------------------------------------------------------------------------";
```

File: main.cpp
Comment (OrneryCash) [inline main.cpp:1]: Use this main.cpp or Importedmain.cpp as a main file?

Comment (zepdon) [reply main.cpp:1]: I push as a new file(Importedmain.cpp) in case you want to keep the old file(main.cpp)

Comment (OrneryCash) [reply main.cpp:1]: okie

**Review by OrneryCash — APPROVED** [review]
lgtm

**Pull request merged**